server.js

```diff
@@ -1,38 +1,140 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-import mongoose from 'mongoose'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
-mongoose.Promise = Promise
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+import express from 'express';
+import bodyParser from 'body-parser';
+import cors from 'cors';
+import mongoose from 'mongoose';
+
+import bookData from './data/books.json';
+
+// Allowing for us to connect to mongodb database
+// Line 10 url to database which we can use in mongodb to access the server data
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/books";
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
+mongoose.Promise = Promise;
+
+// Creating the model/structure of each of the instances that will be entered into the database. This model is based on the object structure/keys/values defined for the array elements in the books.json data set. We write the key name and the value type of each of the object keys as defined in the books.json.
+const Book = mongoose.model('Book', {
+  bookID: Number,
+  title: String,
+  authors: String,
+  average_rating: Number,
+  isbn: String,
+  isbn13: String,
+  language_code: String,
+  num_pages: Number,
+  ratings_count: Number,
+  text_reviews_count: Number,
+});
+
```

Comment:
I understand that you had to define the isbn as a String. But I don't understand why this is. It looks like a number to me :). Maybe the reviewing code coach can explains this?

Reply:
ISBN is the international identification number and it is safer to store it as a string because it has a certain length and when someone makes mistake like this:
ISBN(as a number) + 1 = ISBN of different book.
ISBN(as a string) + 1 = ISBN1 -> error: string is too long